en/config/motors.md

```diff
@@ -0,0 +1,12 @@
+# Motor Checks
+
+After the airframe is configured and the vehicle is setup, it is necessary to validate the motor assignment, spin direction and servo response.
+
+1. First, remove all propellers, switch to Manual flight mode and arm the vehicle.
+1. Increase the throttle a bit, so the vehicle does not automatically disarm. At the same time check that the motors are responding to throttle changes.
+1. Then check that all motors turn to the right direction according to the configured [airframe](../airframes/airframe_reference.md). If one or more spin the wrong direction, they need to be reversed. There are several options to achieve that:
+   1. Swap 2 of the 3 motor cables (it does not matter which ones). If they are not connected via bullet-connectors, re-soldering is required. Therefore the next option is preferable.
+   1. Reverse the direction via [DShot commands](../peripherals/dshot.md#commands) if the ESCs support DShot. DShot should be preferred over PWM in general, as it has further advantages.
```

Comment:
It might be worth mentioning that even when not using DShot it's possible to reverse the motor direction without soldering. E.g. BLHeli ESCs can be configured though the 1 wire interface and their direction can be changed, even without DShot. You can either use a USB linker or a flight controller that supports FC passthrough (like betaflight). The latter is convenient because you can configure all 4 ESCs at the same time (so you can set the same beep volume etc for all of them).

Reply:
Yes I'm aware of the passthrough, but I wasn't sure if it's worth mentioning since PX4 does not support it. That would be convenient as well though.
I didn't know about the USB linker. Is it something easy to find and buy?

Reply:
> Is it something easy to find and buy?

Actually, not really. You will find them on hobbyking etc, but it's almost impossible to find out if it will work with your ESC (depends on the MCU), so it's a bit of trial and error. The demand for them seems to be quite low, any guide online will only suggest FC passthrough.

Reply:
IMO not worth mentioning as the process of getting passthrough working likely to be similar effort to re-soldering and just makes this very complicated. Marking as resolved on that basis.